__tests__/app.test.js

```diff
@@ -21,35 +21,74 @@ describe('/api/topics', () => {
           })
         })
     })
-})
-
-describe("error handling", () => {
-    test("404 for missing endpoints", () => {
+    test('GET 404: should respond with a 404 not found if given an endpoint that does not exist', () => {
         return request(app)
         .get('/api/missing')
         .expect(404)
-        .then(({body}) => {
-        expect(body.msg).toEqual('/api/missing endpoint not found')
-        })
-    })
+        .then((response) => {
+            expect(response.body.msg).toBe("Endpoint does not exist");
+        });
+    });
 })
 
-describe('/api endpoint', () => {
-    test('GET: 200 should return an object describing all available endpoints', () => {
-      return fs
-        .readFile(`${__dirname}/../endpoints.json`, 'utf8')
-        .then((body) => {
-          const expected = JSON.parse(body)
-          return Promise.all([request(app).get('/api').expect(200), expected])
-        })
-        .then((res) => {
-          const { body } = res[0]
-          const expected = res[1]
-          expect(body.endpoints).toEqual(expected)
+// describe("error handling", () => {
+//     test("404 for missing endpoints", () => {
+//         return request(app)
+//         .get('/api/missing')
+//         .expect(404)
+//         .then(({body}) => {
+//         expect(body.msg).toEqual('/api/missing endpoint not found')
+//         })
+//     })
+// })
+
+describe("GET /api", () => {
+    test("GET 200: returns an object describing all the available endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+            const body = response.body;
+            expect(body).toEqual(endpoints);
         })
     })
 })
 
-
-
+describe('/api/articles/:article_id', () => {
+    test("GET: 200 should return the article identified by the specified article_id", () => {
+      const expectedArticle = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
```

Comment:
You _might_ have issues with the created_at as you reseed your database every time you run a test. You would be ok to say that the property exists.